frontend/app/utils/env.server.ts

```diff
@@ -181,7 +181,9 @@ export function getPublicEnv() {
  */
 export function featureEnabled(feature: FeatureName) {
   const { ENABLED_FEATURES } = getEnv();
-  return ENABLED_FEATURES.includes(feature);
+  if (!ENABLED_FEATURES.includes(feature)) {
+    throw new Response('Not Found', { status: 404 });
+  }
 }
```

Comment:
I would modify this implementation slightly (update jsdoc; log a warning)..

```typescript
const log = getLogger('env.server');

/**
 * A helper function to be used in actions and loaders that checks if a feature
 * is enabled. If the feature is not enabled, a 404 response will be thrown.
 */
export function featureEnabled(feature: FeatureName) {
  const { ENABLED_FEATURES } = getEnv();

  if (!ENABLED_FEATURES.includes(feature)) {
    log.warn('Feature [%s] is not enabled; returning 404 response', feature);
    throw new Response(null, { status: 404 });
  }
}
```

Reply:
**Reason:** it might be helpful to future devs when they hit a URL and get a 404.

Reply:
Also: I removed the response body.. Browsers understand a 404, so there's no need to send a 'Not Found' response body.

Reply:
Makes sense. Implemented in commit 6278670f1110c2b2127cd93ac3535624f55c6078.